vault/notebook/integrity.md

```diff
@@ -0,0 +1,54 @@
+## Integrity
```

Comment:
This really should be a first level title i.e. `# Integrity`

Reply:
Actioned: https://github.com/life-itself/community/pull/202/commits/796e1c9f27751ba2e28aa3726366e9a4dbba1939